desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/takeoffer/TakeOfferView.java

```diff
@@ -336,11 +336,9 @@ protected void activate() {
             tradeFeeInBsqToggle.setManaged(false);
             buyBsqBox.setVisible(false);
             buyBsqBox.setManaged(false);
-        }
-
-        if (!model.isShowBuyBsqHint()) {
```

Comment:
I see similar code fragment in [MutableOfferView](https://github.com/bisq-network/bisq/blob/00dc7998ef3e77496605def8111eae8a91b7faae/desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/MutableOfferView.java#L277-L280) - shouldn't it be updated similarly?

Reply:
It works there as the listener is triggered, but yes I guess it is better to have the same code there as well. I'll push an additional commit.